lms/templates/header/navbar-authenticated.html

```diff
@@ -52,6 +52,14 @@
           </a>
       </div>
     % endif
+    % if user.is_staff:
+    <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
+      <a class="tab-nav-link discover-new-link" href="/admin_dashboard"
+         aria-current="${'page' if '/courses' in request.path else 'false'}">
+         ${_('Reports')}
+      </a>
+    </div>
```

Comment:
i mean inside the `if` condition should have indentations.

Reply:
but the same line is present inside many existing divs and its not indented there, line 50 for example. shouldn't it be consistent with rest of the script?